common/lib/xmodule/xmodule/modulestore/split_mongo/split_draft.py

```diff
@@ -500,7 +493,7 @@ def import_xblock(self, user_id, course_key, block_type, block_id, fields=None,
                 draft_course = course_key.for_branch(ModuleStoreEnum.BranchName.draft)
                 with self.branch_setting(ModuleStoreEnum.Branch.draft_preferred, draft_course):
                     draft_block = self.import_xblock(user_id, draft_course, block_type, block_id, fields, runtime)
-                    return self.publish(draft_block.location.version_agnostic(), user_id, blacklist=EXCLUDE_ALL, **kwargs)
+                    # Don't publish the draft block here. Instead create the published block below.
```

Comment:
this is also confusing to me. Why were we publishing drafts during imports in the first place?